lib/ex_machina/ecto.ex

```diff
@@ -66,6 +70,34 @@ defmodule ExMachina.Ecto do
     |> drop_ecto_fields
   end
 
+  @doc """
+  Same as `params_for/2`, but it inserts all associations and sets the
+  foreign_keys.
+
+  ## Example
+
+      def article_factory do
+        %MyApp.Article{title: "An Awesome Article", author: build(:user)}
+      end
+
+      # Inserts an author and returns
+      # %{title: "An Awesome Article", author_id: 12}
+      params_with_assocs(:article)
+  """
```

Comment:
It might be worth adding a note to `params_for/2` that belongs_to associations are not included and they should look at `params_with_assocs/2` if they need it. Otherwise I'm not sure people will realize it's here or why it matters